src/CMakeLists.txt

```diff
@@ -65,9 +80,8 @@ install(TARGETS mgclient-static mgclient-shared
         LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR})
 install(DIRECTORY
         "${CMAKE_SOURCE_DIR}/include/"
-        DESTINATION
-        ${CMAKE_INSTALL_INCLUDEDIR})
+        DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 install(FILES
         "${CMAKE_CURRENT_BINARY_DIR}/mgclient-export.h"
-        DESTINATION
-        ${CMAKE_INSTALL_INCLUDEDIR})
+        "${PROJECT_SOURCE_DIR}/include/mgclient.h"
```

Comment:
Isn't this covered with the install above?

Reply:
Yea, I don't remember why I did this... will check and fix...